### ISSUE TITLE
MENGA DECHIL A MALK

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#7]
Is this a rude word for someone of Chamarro descent?

**Comment by Gnalu** [comment]
Very. I also have heard ‘Menga titidas’ but that one is subtle and in the
context it is said to a person like chad ra marialas to describe a person.

 My wondering is, are Belauans only people that would ask ‘where a person
is from?’ I.e ‘ng chad er ker?’

**Comment by Lukes-Imeyuns** [comment]
Yes, to me it is very rude. I would probably do/say something bad if
someone told me I ate chicken poop!

I wonder if because we're such a small place and everyone is related,
whether by blood or clan affiliation, we always want to know where a person
is from. And the next step is to ask who their parents are. Perhaps we want
to know if they're a relative....? Idk.

In the US, I have experienced that "Where are you from?" is not always
polite or politically correct, think microaggression...

**Comment by s-o-u-a-n-g** [comment]
Microagression for sure.

There's other dismissive and diminutive terms that have developed in our vocabulary like calling other Micronesians "chad er a iungs" even though we clearly are also islanders. 

Being othered by the colonizers really did a number on the culture and the people so that we have so many terms and behaviors that diminish the humanity of our fellow Austronesian distant cousins.

**Comment by smith-371** [comment]
>  My wondering is, are Belauans only people that would ask ‘where a person is from?’ I.e ‘ng chad er ker?’

It is normal for almost all of the world. In any place I have visited before I was asked 'where are you from?' with no agression or offence meant.

**Comment by s-o-u-a-n-g** [comment]
It's all in the tone, cadence, inflection, body language, and facial expression. We have to remember that foreigners are often viewed with suspicion, and rightfully so, given /gestures broadly at the last 135 years// our history.

**Comment by Lukes-Imeyuns** [comment]
"Where are you from?"

"Palau."

"Huh. WHAT is that?"

"It's an island nation in the Western Pacific. You did ask WHERE I was
from, right?"

"Island? How did you get here? Airplane?"

"Nope. Bamboo raft."

Yep! I had this exchange with a college classmate. Of course I knew the
person enough to joke like that.
